src/muxer/muxer_pass.c

```diff
@@ -446,10 +447,34 @@ pass_muxer_reconfigure(muxer_t* m, const struct streaming_start *ss)
     pm->pm_dst_onid  = pm->pm_src_onid;
   }
   pm->pm_pmt_pid     = ss->ss_pmt_pid;
+  pm->pm_rewrite_pmt = !!pm->m_config.u.pass.m_rewrite_pmt;
   pm->pm_rewrite_sdt = !!pm->m_config.u.pass.m_rewrite_sdt;
   pm->pm_rewrite_nit = !!pm->m_config.u.pass.m_rewrite_nit;
   pm->pm_rewrite_eit = !!pm->m_config.u.pass.m_rewrite_eit;
 
+  if (pm->pm_pmt_pid == DVB_SDT_PID && pm->pm_rewrite_pmt) {
+    tvhwarn(LS_PASS, "PMT PID shared with A/V, rewrite disabled");
+    pm->pm_rewrite_pmt = 0;
+  }
+  if (pm->pm_pmt_pid == DVB_SDT_PID && pm->pm_rewrite_sdt) {
+    tvhwarn(LS_PASS, "PMT PID shared with A/V, rewrite disabled");
+    pm->pm_rewrite_pmt = 0;
+    tvhwarn(LS_PASS, "SDT PID shared with A/V, rewrite disabled");
+    pm->pm_rewrite_sdt = 0;
+  }
+  if (pm->pm_pmt_pid == DVB_NIT_PID && pm->pm_rewrite_nit) {
+    tvhwarn(LS_PASS, "PMT PID shared with A/V, rewrite disabled");
+    pm->pm_rewrite_pmt = 0;
+    tvhwarn(LS_PASS, "NIT PID shared with A/V, rewrite disabled");
+    pm->pm_rewrite_nit = 0;
+  }
```

Comment:
Turn off NIT rewrite only.

Reply:
This is something I have seen already before: only if I disable both, PMT and NIT rewrite, the PMT with `pm->pm_pmt_pid == DVB_SDT_PID` survives. Turning off NIT alone does not work.